src/apic-extension/HISTORY.rst

```diff
@@ -5,7 +5,15 @@ Release History
 
 1.0.0b5
 +++++
+* Add: Support yaml file for `az apic api register` command
+* Update: Update some command names, parameter names as well as related descriptions for better understanding. Please leverage `-h` option or refer Azure CLI reference doc to see full list of commands and parameters.
+* Update: Some parameters now have constraints to ensure valid values are provided.
```

Comment:
@* Update: Introduction to parameter constraints to ensure that valid values are provided.